webapp/channels/src/components/advanced_text_editor/formatting_bar/formatting_icon.tsx

```diff
@@ -129,7 +130,7 @@ const FormattingIcon = (props: FormattingIconProps): JSX.Element => {
     const bodyAction = (
         <IconContainer
             type='button'
-            id={`FormattingControl_${mode}`}
+            id={props.id || `FormattingControl_${mode}`}
```

Comment:
Fixes issue of `Keyboard focus is moving out of list option of message priority popup on background controls` where "Message priority" control button at ATE is being skipped on tab .

<img width="1323" alt="Screenshot 2023-08-08 at 6 42 06 AM" src="https://github.com/mattermost/mattermost/assets/5334504/8761a00a-d4a0-4f96-a954-9ef7869526cc">